meta/api/endpoint/ApiEndPoint.ts

```diff
@@ -19,6 +19,17 @@ export const ApiEndPoint = {
 
   CycleData: {
     descriptions: () => apiPath('cycleData', 'descriptions'),
+
+    OriginalDataPoint: {
+      one: () => apiPath('cycleData', 'originalDataPoints', 'originalDataPoint'),
+
+      // Used for print view
+      many: () => apiPath('cycleData', 'originalDataPoints'),
```

Comment:
printView fetches many originalDataPoints (all infos) and displays them before ODP tables